js/ui/handler/drag_rotate.js

```diff
@@ -200,7 +200,14 @@ class DragRotateHandler {
         } else {
             const buttons = (e.ctrlKey ? 1 : 2),  // ? ctrl+left button : right button
                 button = (e.ctrlKey ? 0 : 2);   // ? ctrl+left button : right button
-            return (e.type === 'mousemove' ? e.buttons & buttons === 0 : e.button !== button);
+            let eventButton = e.button;
+            if (typeof InstallTrigger !== 'undefined' && e.button === 2 && e.ctrlKey) {
```

Comment:
Should this be special-cased for the Mac? As far as I know, this issue only occurs on the Mac, where control-clicking and long-clicking were historically the ways to bring up the context menu on a one-button mouse. 🖱